### ISSUE TITLE
🐛 RangeError: Port should be >= 0 and < 65536. `VSCode`

### ISSUE THREAD

**Issue by hyoretsu** [rome/tools#3933]
### Environment information

```block
CLI:
  Version:              10.0.1
  Color support:        true

Platform:
  CPU Architecture:     x86_64
  OS:                   linux

Environment:
  ROME_LOG_DIR:         unset
  NO_COLOR:             unset
  TERM:                 "xterm-256color"

Rome Configuration:
  Status:               loaded
  Formatter disabled:   false
  Linter disabled:      false

Workspace:
  Open Documents:       0

Discovering running Rome servers...

Running Rome Server: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ The client isn't connected to any server but rage discovered this running Rome server.

Server:
  Version:              10.0.1
  Name:                 rome_lsp
  CPU Architecture:     x86_64
  OS:                   linux

Workspace:
  Open Documents:       0

Rome Server Log:

⚠ Please review the content of the log file before sharing it publicly as it may contain sensitive information:
  * Path names that may reveal your name, a project name, or the name of your employer.
  * Source code

├─12046136ms INFO rome_lsp::server Starting Rome Language Server...
├─12057532ms INFO rome_lsp::server Starting Rome Language Server...
```

### What happened?

When I stay too much time in a workspace with Rome (in this case, hibernated my PC) or close it and come back again, I receive the following error:

```
Rome client: couldn't create connection to server.
RangeError: Port should be >= 0 and < 65536. Received .
```

### Expected result

It shouldn't throw that error.

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by hyoretsu** [comment]
*Might've been fixed in a preview release, if so I'll close this.

**Comment by leops** [comment]
This should have been fixed with #3957, or at least the extension should now print a more detailed error report for what went wrong
